examples/tutorials/05_ordinary_models.ipynb

```diff
@@ -0,0 +1,381 @@
+{
```

Comment:
<p>What is that for?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/analysiscenter/batchflow/pull/383/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='23'/>